testData/solutions/win32.json5

```diff
@@ -5797,6 +6462,498 @@
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons": "SwapMouseButtonsConfig.value"
+                }
+            },
+            "configureScrollWheelMode": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETWHEELSCROLLLINES",
+                    "setAction": "SPI_SETWHEELSCROLLLINES",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "ScrollWheelModeConfig": {
+                        "schema": {
+                            "title": "Mouse scroll wheel mode",
+                            "description": "Changes the way scroll wheel behaves, between lines and screens",
+                            "type": "number",
+                            "default": 3,
+                            "min": 0,
+                            "max": 25,
+                            "divisibleBy": 1,
+                            // This special value trigger the special meaning
+                            // of full-page scrolling.
+                            "specialValue": "'full-page' | 4294967295"
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "ScrollWheelModeConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "defaultInputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines",
+                            "defaultOutputPath": "value",
+                            "match": {
+                                "full-page": 4294967295,
+                                "25": 4294967295
+                            },
+                            "noMatch": {
+                                "outputValue": {
+                                    "transform": {
+                                        "type": "fluid.transforms.condition",
+                                        "condition": {
+                                            "transform": {
+                                                "type": "fluid.transforms.inRange",
+                                                "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.ScrollLines",
+                                                "min": 1,
+                                                "max": 25
+                                            }
+                                        },
+                                        "false": 3,
```

Comment:
Why do we output 3 when the value is greater than 25, rather than the maximum range value?